src/routes/_layout.svelte

```diff
@@ -3,17 +3,16 @@
   import Footer from "@/components/sections/footer.svelte";
 
   export let segment: string;
-  const pageTitle = segment === undefined ? "ACM at CSUF" : segment;
 </script>
 
 <Navbar segment="{segment}" />
-<main><slot /></main>
+<main style="min-height: 100vh;"><slot /></main>
```

Comment:
These styles would look good between some of these: `<style></style>`.

Reply:
ok